api/desecapi/tests/test_rrsets.py

```diff
@@ -197,18 +197,49 @@ def test_create_other_rr_sets(self):
         response = self.client.post_rr_set(self.other_domain.name, **data)
         self.assertStatus(response, status.HTTP_404_NOT_FOUND)
 
+    def test_create_my_rr_sets_chunk_too_long(self):
+        def _create_data(record):
+            return {'records': [f'{record}'], 'ttl': 3600, 'type': 'TXT', 'subname': f'name{len(record)}'}
+
+        with self.assertPdnsRequests(self.requests_desec_rr_sets_update(self.my_empty_domain.name)):
+            response = self.client.post_rr_set(
+                self.my_empty_domain.name,
+                # record of length 500 bytes in chunks of max 255 each (RFC 4408)
+                # "AAA...AAA" "AAA...AAA"
+                **_create_data('"' + 'A' * 255 + '"')
+            )
+            self.assertStatus(response, status.HTTP_201_CREATED)
+
+        response = self.client.post_rr_set(
+            self.my_empty_domain.name,
+            # record of length 501 bytes in chunks of max 255 each (RFC 4408)
+            # "AAA...AAA" "AAA...AAA"
+            **_create_data('"' + 'A' * 256 + '"')
+        )
+        self.assertStatus(response, status.HTTP_400_BAD_REQUEST)
+        self.assertIn('Splitting the data in chunks is required (RFC 4408).', str(response.data))
+
     def test_create_my_rr_sets_too_long_content(self):
-        def _create_data(length):
-            content_string = 'A' * length
-            return {'records': [f'{content_string}'], 'ttl': 3600, 'type': 'TXT', 'subname': f'name{length}'}
+        def _create_data(record):
+            return {'records': [f'{record}'], 'ttl': 3600, 'type': 'TXT', 'subname': f'name{len(record)}'}
 
         with self.assertPdnsRequests(self.requests_desec_rr_sets_update(self.my_empty_domain.name)):
-            response = self.client.post_rr_set(self.my_empty_domain.name, **_create_data(255))
+            response = self.client.post_rr_set(
+                self.my_empty_domain.name,
+                # record of length 500 bytes in chunks of max 255 each (RFC 4408)
+                # "AAA...AAA" "AAA...AAA"
+                **_create_data('"' + 'A' * 255 + '" "' + 'A' * 240 + '"')
+            )
             self.assertStatus(response, status.HTTP_201_CREATED)
 
-        response = self.client.post_rr_set(self.my_empty_domain.name, **_create_data(256))
+        response = self.client.post_rr_set(
+            self.my_empty_domain.name,
+            # record of length 501 bytes in chunks of max 255 each (RFC 4408)
+            # "AAA...AAA" "AAA...AAA"
+            **_create_data('"' + 'A' * 255 + '" "' + 'A' * 241 + '"')
```

Comment:
Is this all there is to the tokenization? If so, let's our API handle it automatically.

Also, `**_create_data(f'"{"A"*255}" "{"A"*241}"')`

Reply:
Please elaborate :) Not sure how/what the API should be doing precisely. If, e.g., a token exceeds the max. length, then what do we do?